functions/v2/http-logging/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_structured_logging]
+const cloudLogging = require('@google-cloud/logging');
+const functions = require('@google-cloud/functions-framework');
+
+functions.http('structured_logging', async (req, res) => {
```

Comment:
In the CLoud Logging screenshot you provided you can see the trace field has `...{{projectId}}..` which means the project id isn't set. You will need to add `await logging.setProjectId()`